### PR TITLE
Correct prism languages so they all exist

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Random Prism tags which were typos or wrong. Now corrected.

### CODE REVIEW

File: product_docs/docs/pem/9/pem_online_help/pem_worker_usage_guide.mdx [hunk 1]
```diff
@@ -9,251 +9,251 @@ The PEM Worker performs tasks on behalf of the PEM Agent. Please note that all t
...
-    remove-webhook-options:
-      --webhook-name (Name of the webhook endpoint.)
-
-      NOTE:
-       Set environment variable PEM_SERVER_PASSWORD to provide the password for the
-       PEM server to allow the pemworker to connect as a PEM admin user.
-    ```
+- On Linux:  
+
+```shsession
```
Comment (josh-heyer) [inline product_docs/docs/pem/9/pem_online_help/pem_worker_usage_guide.mdx:14]: lost the indentation - that drops these out of the list. Arguably these should be sections with headings, not list items - but for now they are. Suggest either changing them to headings, or re-adding indentation.

File: product_docs/docs/pem/8/pem_online_help/pem_worker_usage_guide.mdx [hunk 2]
```diff
@@ -9,155 +9,154 @@ The PEM Worker performs tasks on behalf of the PEM Agent. Please note that all t
...
+    --config-file/-c <config_file> (Path to the agent configuration file.)
+
+    NOTE:
+      Set environment variable PEM_SERVER_PASSWORD to provide the password for the
+      PEM server to allow the pemworker to connect as a PEM admin user.
+```
+
+##  On Windows:  
+
+    ```shell-session
```
Comment (josh-heyer) [inline product_docs/docs/pem/8/pem_online_help/pem_worker_usage_guide.mdx:159]: Drop the indentation here now that these are in sections and not a list

**Review by josh-heyer — CHANGES_REQUESTED** [review]
Couple of nits

**Comment by drothery-edb** [comment]
@josh-heyer do the Kubernetes fixes need to be made in the source repo?

**Comment by josh-heyer** [comment]
> @josh-heyer do the Kubernetes fixes need to be made in the source repo?

Good catch @drothery-edb ! 

@djw-m most of the kubernetes changes should be made here: https://github.com/cloudnative-pg/cloudnative-pg/tree/main/docs/src ...with the exception of openshift, which is here: https://github.com/EnterpriseDB/cloud-native-postgres/blob/main/docs/src/openshift.md

(these can be committed as well, but they'll be overwritten next release if not done upstream)

**Comment by josh-heyer** [comment]
...miiight be easier to just add `console` as an alias here: https://github.com/EnterpriseDB/docs/blob/develop/gatsby-config.js#L292

File: product_docs/docs/pem/8/pem_online_help/pem_worker_usage_guide.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pem/8/pem_online_help/pem_worker_usage_guide.mdx:159]: Reworked into H3/H4 sections for better readability.

**Comment by djw-m** [comment]
Looking upstream, I see mkdocs uses pygments where console is alias for...
shell-session. In which case we should add the alias.... reverting the
changes on the Kubernetes docs, adding the alias.

**Comment by drothery-edb** [comment]
@josh-heyer do you want to take another look at this or can we merge it?

**Pull request merged**